vibration/ios/Classes/VibrationPluginSwift.swift

```diff
@@ -176,7 +176,11 @@ public class VibrationPluginSwift: NSObject, FlutterPlugin {
                 result(true)
                 return
             }
-
+            guard let pattern = myArgs["pattern"] as? [Int] else {
```

Comment:
In this case the device will vibrate once if the pattern argument is not provided. For example, if you only provide `duration`.